WISE-Community/WISE
# Open milestone report from step grading

To test:
- Make sure milestone reports function properly from the milestones section of the Classroom Monitor.
- Navigate to the step grading view for steps that have linked milestone reports.
- Make sure the "Report" button opens the milestone details dialog and shows the report content and recommendations. (The Student Work tab should be hidden from this view.)

Close #2299.
